modules/protect/math-fallback.php

```diff
@@ -5,9 +5,19 @@
 	 * The math captcha fallback if we can't talk to the Protect API
 	 */
 	class Jetpack_Protect_Math_Authenticate {
+		
+		static $loaded;
 
 		function __construct() {
+			
+			if ( self::$loaded ) {
+				return;
+			}
+			
+			self::$loaded = 1;
+			
 			add_action( 'login_form', array( $this, 'math_form' ) );
+			
 			if( isset( $_POST[ 'jetpack_protect_process_math_form' ] ) ) {
 				add_action( 'init', array( $this, 'process_generate_math_page' ) );
 			}
```

Comment:
When someone logins in successfully after seeing the Math, can we remove their block all together?